test/teos/e2e/test_cli_e2e.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import json
+from time import sleep
+
+from contrib.client import teos_client
+
+from common.exceptions import InvalidParameter
+from common.cryptographer import Cryptographer
+
+from teos.cli.rpc_client import RPCClient
+
+from test.teos.conftest import (
+    create_txs,
+    generate_block_with_transactions,
+    generate_blocks,
+    config,
+)
+from test.teos.e2e.conftest import build_appointment_data
+
+teos_base_endpoint = "http://{}:{}".format(config.get("API_BIND"), config.get("API_PORT"))
+
+user_sk = Cryptographer.generate_key()
+user_id = Cryptographer.get_compressed_pk(user_sk.public_key)
+
+
+@pytest.fixture
+def rpc_client():
+    return RPCClient(config.get("RPC_BIND"), config.get("RPC_PORT"))
+
+
+def get_appointment_info(teos_id, locator, sk=user_sk):
+    sleep(1)  # Let's add a bit of delay so the state can be updated
+    return teos_client.get_appointment(locator, sk, teos_id, teos_base_endpoint)
+
+
+def add_appointment(teos_id, appointment_data, sk=user_sk):
+    return teos_client.add_appointment(appointment_data, sk, teos_id, teos_base_endpoint)
+
+
+def test_get_all_appointments(teosd, rpc_client):
+    _, teos_id = teosd
+
+    # Check that there is no appointment, so far
+    all_appointments = json.loads(rpc_client.get_all_appointments())
+    watching = all_appointments.get("watcher_appointments")
+    responding = all_appointments.get("responder_trackers")
+    assert len(watching) == 0 and len(responding) == 0
+
+    # Register a user
+    available_slots, subscription_expiry = teos_client.register(user_id, teos_id, teos_base_endpoint)
```

Comment:
The return values are never used

Reply:
Removed unused variables in 37ef745fe6a74b3cdad0a95c96b029c4ec59e57f.